### ISSUE TITLE
See in the build log time spent on each command

### ISSUE THREAD

**Issue by trexinc** [appveyor/ci#2769]
It would be really nice to see in the build log the time spent on each command, this would help to understand which parts of the build process need to be improved. Travis CI has this feature.

**Comment by IlyaFinkelshteyn** [comment]
Yes, we plan to add an option to collapse log for steps and see it's summary. However note that now you can hover a mouse over line number and see the time for each specific line.

**Comment by trexinc** [comment]
Thanks, didn't notice the "hover" feature before.
